__tests__/helpers.js

```diff
@@ -26,90 +27,90 @@ describe('helpers', () => {
       describe('date is a date type', () => {
         it('should return date as it is', () => {
           const date = new Date(2018, 0, 1);
-          expect(helpers.parseISO(date)).to.equal(date);
+          expect(helpers.parseISO(date)).toEqual(date);
         });
       });
 
       describe('date is a string type', () => {
         it('should return date as date type', () => {
           const date = '2018-01-01';
-          expect(helpers.parseISO(date)).to.deep.equal(new Date(date));
+          expect(helpers.parseISO(date)).toEqual(new Date(date));
         });
       });
 
       describe('date is a integer type', () => {
         it('should return date as it is', () => {
           const date = 1514764800000; // 2018-01-01
-          expect(helpers.parseISO(date)).to.equal(date);
+          expect(helpers.parseISO(date)).toEqual(new Date(date));
```

Comment:
The result changed? Why though? 

The `it` description needs to change too.

Reply:
Because `parseISO` is supposed to convert to `Date` right? Previously timestamp was kept untouched

Reply:
Oh okay, could you update the `it` description to `should return date as date type`?